src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random RANDOM = new Random();
+    private static final String DEFAULT_COLOR = "white";
+    private static final int DEFAULT_RADIUS = 10;
+
+    public Figure getRandomFigure() {
+        int figurType = RANDOM.nextInt(5);

```

Comment:
The variable name 'figurType' has a typo and does not follow Java naming conventions. Consider renaming it to 'figureType' to improve code readability.